main.go

```diff
@@ -7,15 +7,29 @@ import (
 	"io"
 	"net/http"
 	"os"
+	"strings"
 )
 
+type stringArray []string
```

Comment:
Why not using a `map[string][]string` instead? Also the type name, while correct, is too generic, maybe `headersSomething`?

Reply:
Changed to MIMEHeader from textproto which is map[string][]string. Also renamed. I put that name before because I thought of this as a generic string array argument parser. But this makes more sense.